src/main/java/org/jabref/gui/exporter/ExportToClipboardAction.java

```diff
@@ -135,4 +148,14 @@ private String readFileToString(Path tmp) throws IOException {
             return reader.lines().collect(Collectors.joining(OS.NEWLINE));
         }
     }
+
+    private class ExportResult {
+        final String content;
```

Comment:
not sure about java conventions, but shouldn't be variables in a class always be hidden by get/set methods?

Reply:
More or less, yes. However, as this is a private inner class you can omit it.